<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>515</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>515</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qwitter Book - A Qt-based Twitter and Facebook Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/images.qrc">
    <normaloff>:/twitter/TwitterPostIcon.png</normaloff>:/twitter/TwitterPostIcon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>236</red>
         <green>233</green>
         <blue>216</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QLabel" name="twitterBgLogo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>110</y>
      <width>210</width>
      <height>49</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>210</width>
      <height>49</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>210</width>
      <height>49</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/Twitter.png</pixmap>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidgetTweetWindow">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>515</width>
      <height>256</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>236</red>
          <green>233</green>
          <blue>216</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>236</red>
          <green>233</green>
          <blue>216</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>236</red>
          <green>233</green>
          <blue>216</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>236</red>
          <green>233</green>
          <blue>216</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabPublicTimeline">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string>Right now you are walking through the Public Tweet-Space-Time Continuum</string>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Public Timeline</string>
     </attribute>
     <widget class="QScrollArea" name="scrollAreaPublicTimeline">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>510</width>
        <height>236</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContentsPT">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>508</width>
         <height>234</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QTextBrowser" name="textBrowserPT">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>508</width>
          <height>234</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFriendsTimeline">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string>Enjoy the stroll in the Tweet-Spacetime with your friends!!</string>
     </property>
     <attribute name="title">
      <string>Friends Timeline</string>
     </attribute>
     <widget class="QScrollArea" name="scrollAreaFriendsTimeline">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>510</width>
        <height>236</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContentsFT">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>508</width>
         <height>234</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QTextBrowser" name="textBrowseFT">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>508</width>
          <height>234</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMyTweetsTimeline">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string>Oh! So now you wish to go into the reverie within your own Tweet(Multi)verse</string>
     </property>
     <attribute name="title">
      <string>My Tweets</string>
     </attribute>
     <widget class="QScrollArea" name="scrollAreaMyTweetsTimeline">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>510</width>
        <height>236</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContentsMTT">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>508</width>
         <height>234</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QTextBrowser" name="textBrowserMTT">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>508</width>
          <height>234</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>515</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTimeline">
    <property name="statusTip">
     <string>View various Twitter Timelines</string>
    </property>
    <property name="title">
     <string>Timeline</string>
    </property>
    <addaction name="actionPublic"/>
    <addaction name="actionFriends"/>
    <addaction name="actionOwn"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionIs_It_Working"/>
    <addaction name="actionAbout_Qt_Twitter"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuOther_Services">
    <property name="title">
     <string>Other Services</string>
    </property>
    <widget class="QMenu" name="menuSet_FaceBook_Status">
     <property name="title">
      <string>Set FaceBook Status</string>
     </property>
     <addaction name="actionFacebook_Status_Twitter"/>
    </widget>
    <addaction name="menuSet_FaceBook_Status"/>
    <addaction name="actionView_Images_From_Area"/>
   </widget>
   <widget class="QMenu" name="menuTwitter">
    <property name="title">
     <string>Twitter</string>
    </property>
    <addaction name="actionSet_New_Status"/>
    <addaction name="actionFollow"/>
    <addaction name="actionLeave"/>
    <addaction name="actionUser_Methods"/>
    <addaction name="actionDirect_Message_Methods"/>
    <addaction name="actionFriendship_Methods"/>
    <addaction name="actionArchive"/>
    <addaction name="actionFavorite_Methods"/>
    <addaction name="actionBlock_Methods"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTimeline"/>
   <addaction name="menuTwitter"/>
   <addaction name="menuOther_Services"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="actionSet_New_Status"/>
   <addaction name="actionFollow"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionLogin">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/user-online.svg</normaloff>
     <normalon>:/twitter/twitter_system_tray1.png</normalon>:/twitter/user-online.svg</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="toolTip">
    <string>Login To Twitter</string>
   </property>
   <property name="statusTip">
    <string>Simple Login to twitter</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLogout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/user-offline.svg</normaloff>:/twitter/user-offline.svg</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="toolTip">
    <string>Logout From Twitter</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/print.jpeg</normaloff>:/twitter/print.jpeg</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionPublic">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/twitter_system_tray_2.0.png</normaloff>:/twitter/twitter_system_tray_2.0.png</iconset>
   </property>
   <property name="text">
    <string>Public</string>
   </property>
   <property name="statusTip">
    <string>Display Public TimeLine From Twitter Server</string>
   </property>
  </action>
  <action name="actionFriends">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/twitter_system_tray.png</normaloff>:/twitter/twitter_system_tray.png</iconset>
   </property>
   <property name="text">
    <string>Friends</string>
   </property>
   <property name="statusTip">
    <string>Display Friends' TimeLine From Twitter Server</string>
   </property>
  </action>
  <action name="actionOwn">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/twhitter_btn_02.png</normaloff>:/twitter/twhitter_btn_02.png</iconset>
   </property>
   <property name="text">
    <string>Own</string>
   </property>
   <property name="statusTip">
    <string>Display Your Own TimeLine From Twitter Server</string>
   </property>
  </action>
  <action name="actionAbout_Qt_Twitter">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/TwitterPostIcon.png</normaloff>:/twitter/TwitterPostIcon.png</iconset>
   </property>
   <property name="text">
    <string>About Qt Twitter</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/qt.png</normaloff>:/twitter/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/help-about.svg</normaloff>:/twitter/help-about.svg</iconset>
   </property>
   <property name="text">
    <string>New From Twitter</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/application-exit.svg</normaloff>:/twitter/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFacebook_Status_Twitter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/icon.facebook.png</normaloff>:/twitter/icon.facebook.png</iconset>
   </property>
   <property name="text">
    <string>Facebook Status = Twitter</string>
   </property>
   <property name="toolTip">
    <string>Set Facebook Status equal to Twitter's microblog from next update</string>
   </property>
   <property name="statusTip">
    <string>Set Facebook Status equal to Twitter's microblog from next update</string>
   </property>
   <property name="whatsThis">
    <string>Set Facebook Status equal to Twitter's microblog from next update</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/configure.svg</normaloff>:/twitter/configure.svg</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionSet_New_Status">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/im-status-message-edit.svg</normaloff>:/twitter/im-status-message-edit.svg</iconset>
   </property>
   <property name="text">
    <string>Set New Status</string>
   </property>
  </action>
  <action name="actionFollow">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/add-profile.svg</normaloff>:/twitter/add-profile.svg</iconset>
   </property>
   <property name="text">
    <string>Follow</string>
   </property>
  </action>
  <action name="actionLeave">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/remove-profile.svg</normaloff>:/twitter/remove-profile.svg</iconset>
   </property>
   <property name="text">
    <string>Leave</string>
   </property>
  </action>
  <action name="actionUser_Methods">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/personal.svg</normaloff>:/twitter/personal.svg</iconset>
   </property>
   <property name="text">
    <string>User Methods</string>
   </property>
  </action>
  <action name="actionDirect_Message_Methods">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/message-unread.svg</normaloff>:/twitter/message-unread.svg</iconset>
   </property>
   <property name="text">
    <string>Direct Message Methods</string>
   </property>
  </action>
  <action name="actionFriendship_Methods">
   <property name="text">
    <string>Friendship Methods</string>
   </property>
  </action>
  <action name="actionArchive">
   <property name="text">
    <string>archive</string>
   </property>
  </action>
  <action name="actionFavorite_Methods">
   <property name="text">
    <string>Favorite Methods</string>
   </property>
  </action>
  <action name="actionBlock_Methods">
   <property name="text">
    <string>Block Methods</string>
   </property>
  </action>
  <action name="actionIs_It_Working">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/twitter/questionmark</normaloff>:/twitter/questionmark</iconset>
   </property>
   <property name="text">
    <string>Is It Working?</string>
   </property>
  </action>
  <action name="actionView_Images_From_Area">
   <property name="text">
    <string>View Images From Area</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
